google-cloud-datastore/src/main/java/com/google/cloud/datastore/AggregationResults.java

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.cloud.datastore;
+
+import static com.google.api.client.util.Preconditions.checkNotNull;
+
+import com.google.api.core.BetaApi;
+import com.google.api.core.InternalApi;
+import com.google.cloud.Timestamp;
+import java.util.Iterator;
+import java.util.List;
+import java.util.Objects;
+
+/**
+ * The result of an {@link AggregationQuery} query submission. Contains a {@link
+ * List<AggregationResult>} and readTime {@link Timestamp} in it.
+ *
+ * <p>This can be used to iterate over underlying {@link List<AggregationResult>} directly.
+ *
+ * <p>Though {@link com.google.cloud.datastore.aggregation.CountAggregation} is guaranteed to return
+ * only one {@link AggregationResult} as part of its execution.
+ *
+ * <p>In the future, we might support more complex {@link AggregationQuery} that might result in
```

Comment:
nit - I would remove this to keep the docs just about the class functionality itself

Reply:
Fixed as part of https://github.com/googleapis/java-datastore/commit/a5b462332db2f8b88db4016d2f83046c89e74acd.